### ISSUE TITLE
Feature request: Publish to deno.land/x and nest.land

### ISSUE THREAD

**Issue by nnmrts** [Evercoder/culori#157]
Fortunately culori is already written in modern pure ESM format and AFAIK doesn't use any node.js-specific built-ins, so publishing it to https://deno.land/x and/or https://nest.land should be straight-forward. This would allow Deno users to enjoy culori in a more ergonomic way than currently possible. An example of currently supported ways to use culori in Deno:

```js
import { rgb } from "https://esm.sh/culori@2";
import { rgb } from "https://raw.githubusercontent.com/Evercoder/culori/main/src/index.js";
```

Both of the options above have several drawbacks. `esm.sh` is a great service but usually bundles/transpiles the code of a module, while GitHub `raw.` URLs are really long and also don't really support any form of versioning.

If you agree culori should be made "Deno-community-compatible" and need help with publishing, I'm happy to help. I have gained some experience using Deno for the last few months so far and even published a small module to https://nest.land, so I may be actually useful for once. 😆

---
PS: I just discovered this project while pondering about writing my own "modern" color library which includes `oklab` support. Turns out I don't need to, because culori is already amazing and somehow exactly what I imagined my library to be. Thanks. 🥲

**Comment by nnmrts** [comment]
Just found out that raw GitHub urls don't really work either, for .js files at least. Another alternative is jsdelivr of course, and that's what I'm using for now.

**Comment by danburzo** [comment]
Hi @nnmrts, thanks for the kind words! I've started looking at what we need to change for a first deno-compatible version for [deno.land/x](https://deno.land/x). In particular, decide how to handle mappings in `package.json` for `culori/fn`, `culori/css`, and `culori/all`:

```js
	"exports": {
		"./require": "./bundled/culori.cjs",
		".": "./src/index.js",
		"./css": "./src/bootstrap/css.js",
		"./all": "./src/bootstrap/all.js",
		"./fn": "./src/index-fn.js"
	}
```

(I don't think these are urgent, since they're mostly related to [tree-shaking for a better bundle size](https://culorijs.org/guides/tree-shaking/) and I don't imagine people are using Deno for bundling code for browsers just yet, are they?)

Otherwise, it seems to be pretty straightforward, I'll try a pre-release tag.

**Comment by danburzo** [comment]
Alright, I've pushed the `2.0.1-alpha.0` tag to GitHub, this seems to work pretty well:

```js
import { rgb } from 'https://deno.land/x/culori@v2.0.1-alpha.0/index.js';
console.log(rgb('tomato'));
```

I'll update the docs and publish `2.0.1`, unless there are other concerns to address.

**Comment by nnmrts** [comment]
Going to try this out later today or tomorrow. Thanks for the release!

**Comment by nnmrts** [comment]
@danburzo
> (I don't think these are urgent, since they're mostly related to tree-shaking for a better bundle size and I don't imagine people are using Deno for bundling code for browsers just yet, are they?)

I personally don't, and I also think the future is in bundle-free projects, but for example, currently one of the most popular frameworks out there is this: https://alephjs.org/

Aleph.js bundles the modules into one big file (I think). I assume the overwhelming majority of people using Deno for web development right now, are in fact bundling the code, so publishing `index-fn.js` as well might be particularly useful, at least for the next few years. The thing is though, people can already use that file instead, by just importing it through its url: `https://deno.land/x/culori@v2.0.1-alpha.0/index-fn.js`.

Offering different variants of a module like this is common in the Deno community, so I think this is just fine as it is at the moment.

**Comment by danburzo** [comment]
Starting with the just-published `culori@2.0.1`, the library is available to Deno users. I've also added a bit to [the Getting Started page](https://culorijs.org/getting-started/).
